src/chrome/content/rules/Onlyoffice.com.xml

```diff
@@ -0,0 +1,12 @@
+<!--
+	Host has wildcard DNS and cert.
+-->
+<ruleset name="Onlyoffice.com">
```

Comment:
```suggestion
<ruleset name="Onlyoffice.com (partial)">
```

Reply:
I think we covered all common hosts.

Reply:
Well, there's no way to get all hosts, so I would probably mark as `(partial)`.